src/scripts/domParsers/khanAcademyVideoExtractor.ts

```diff
@@ -0,0 +1,50 @@
+import {VideoExtractor} from "./videoExtractor";
+import {VideoUtils} from "./videoUtils";
+import {YoutubeVideoExtractor} from "./youtubeVideoExtractor";
+
+import {Utils} from "../utils";
+
+export class KhanAcademyVideoExtractor implements VideoExtractor {
+	private youtubeExtractor: YoutubeVideoExtractor;
+
+	constructor() {
+		this.youtubeExtractor = new YoutubeVideoExtractor();
+	}
+
+	public getVideoIds(pageUrl: string, pageContent: string): string[] {
```

Comment:
Do you think we should pipe `pageInfo` through from `clipper.tsx` -> `augmentationHelper.ts` to all  this extraction stuff?